.github/actions/to-integration-tests/Dockerfile

```diff
@@ -15,9 +15,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
-FROM golang:1.14.2-alpine
+FROM golang:1.14.2-buster
```

Comment:
If we're pinning to under-patched go versions for stability, then I don't know if we should change how it's building.

Reply:
This too builds on Alpine if the `musl-dev` package is installed. FWIW, all of the Go ATC components have been built with the [`osusergo`](https://golang.org/doc/go1.11#os/user) and [`netgo`](https://golang.org/doc/go1.5#net) tags ([example](https://github.com/apache/trafficcontrol/pull/4758/files#diff-d57f2a7a8ff4abbc35eef295ce11d1d9R60)) for static linking since #4758, which effectively disables cgo.

Reply:
oh really? I didn't realize, I thought all Go components were built with CGo. I'll definitely look into that, then. I just sort of panicked when I saw "There's no such thing as stdio.h" because generally that means something is very, very wrong. But in the context of Alpine or it actually makes sense to not include the std lib even though you have a compiler. The whole point is to be extremely specific about what winds up in the resulting system.